apps/public-docsite-v9/src/Concepts/Poster/AreaCard.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import { makeStyles, shorthands, tokens, typographyStyles } from '@fluentui/react-components';
+import { AreaInfo, PackageInfo } from './types';
+import { PackageCard } from './PackageCard';
+
+const useStyles = makeStyles({
+  root: {
+    ...shorthands.border(tokens.strokeWidthThick, 'solid', tokens.colorNeutralBackground3),
+    ...shorthands.borderRadius('3px'),
```

Comment:
Big nitpick. Why not use the tokens for radius, border, etc? I know the values you're using here don't 100% align with the tokens but maybe they should?

What do you think?